Test/test_tree.py

```diff
@@ -0,0 +1,57 @@
+from Challenges.tree import Tree
+import unittest
+from Challenges.tree import Node
+
+
+class TestBST(unittest.TestCase):
+    def test_node_initializer(self):
```

Comment:
Unit testing is about testing the public states, behaviors, and interactions of your objects. Testing a constructor doesn't help checking any functionality